src/main/java/com/star_trello/darkside/service/TaskService.java

```diff
@@ -69,11 +88,11 @@
     }
 
     @Transactional
-    public List<User> getObserversList(String[] usernames) {
```

Comment:
I think it is redundant method. This list oftenly contains only 2 usernames. In this method we check non-repeating users. let's go hold Set<String>. and this metod not necessary